cluster/images/etcd-empty-dir-cleanup/Makefile

```diff
@@ -14,9 +14,9 @@
 
 .PHONY:	build push
 
-ETCD_VERSION = 2.2.1
+ETCD_VERSION = 3.0.14
 IMAGE = gcr.io/google_containers/etcd-empty-dir-cleanup
-TAG = 0.0.1
+TAG = 3.0.14
```

Comment:
We are likely to want to rev this faster than etcd versions (e.g. to pick up security fixes in the container base image). Should we make the tag start with the etcd version but have an extension too? e.g. 3.0.14.0